contents.json

```diff
@@ -2043,6 +2043,25 @@
         "http://a2.mzstatic.com/us/r30/Purple1/v4/ae/40/19/ae4019c9-463b-3b73-7147-1b47473596d7/screen696x696.jpeg"
       ]
     },
+    {
+      "title": "Essentia",
+      "category-ids": [
+        "cryptocurrency"
+      ],
+      "tags": [
+        "swift"
+      ],
+      "description": "Multi wallet, All in one Blockchain solution",
+      "license": "gplv3",
```

Comment:
@impul could you update the license to `gpl-3.0`?

Reply:
@vpeschenkov , done